docs/empty-bin.rst

```diff
@@ -12,6 +12,9 @@ individual items that might otherwise be hard to get to.
 Note that if there are liquides in the container, they will empty onto the floor
 and become unusable.
 
+You may also select a stockpile to empty all containers within it, or you can run this
+script when your mouse is over a tile - that will empty all containers present on that tile.
```

Comment:
acting on the cursor under the mouse isn't appropriate for a commandline tool unless it is linked to a hotkey (e.g. `flashstep`). using the keyboard cursor would be better.

If it were to act on the cursor under the mouse, the player would have to:
- move the mouse away from the target square to click on the dfhack logo to bring up the launcher (not everyone does it this way, but this is common)
- move the launcher out of the way of the target tile
- move the cursor back on top of the target tile
- carefully take hands off the mouse so they can type the command
- hit enter

Reply:
hmm yeah fair point. But keyboard cursor is not available in Adventure Mode yet